### PR TITLE
Perf/consuming speed

### PR DESCRIPTION
**Pull request by FlorianSara**
#### Technical checklist

- [x] I tested my code following testing cases.
- [x] I covered my code with automatic tests.
- [x] I added logs in case of error or business logic.
- [x] I well designed my commits.
- [ ] I read my Pull Request's changes.
- [ ] I respected provided design and wording.

#### Pull request's scope

- [ ] Ticket JIRA: [EPSOR-XXX](https://projet-epsor.atlassian.net/browse/EPSOR-XXX)
- [ ] Pull Request's link added as comment to JIRA ticket.
- [ ] Interface screenshots.

- [ ] Add changelog to [Stack Edit](https://stackedit.io/app#providerId=googleDriveWorkspace&folderId=1AKQYihsxhnKiXx9mt7qzxEWU6_YfccPm) when you merge the Pull Request

---

### How to test

1. Launch migrations then seeds
2. Go to ...
3. etc

### CODE REVIEW

File: src/consumer.js [hunk 1]
```diff
@@ -236,15 +239,15 @@ class Consumer {
...
    */
   /* istanbul ignore next */
   async handleMessage(dto, message) {
     const dtoType = dto.constructor.type;
     const handlers = this.handlers[dtoType] || [];
 
     if (handlers.length === 0) {
-      return;
+      return 0;
     }
```
Comment (Pangoraw) [inline src/consumer.js]: inutile en fait

Comment (FlorianSara) [reply src/consumer.js]: ET LE TYPESCRIPT ALORS ?!!!!

Comment (FlorianSara) [reply src/consumer.js]: fixé...

File: src/consumer.js [hunk 2]
```diff
@@ -201,17 +201,20 @@ class Consumer {
...
-              data,
-            });
+            const handlerCount = await this.handleMessage(dto, message);
 
-            this.kafkaConsumer.commitMessageSync(message);
-            logger.info(`Offset committed to Kafka...`, {
-              tags: [this.type, 'consumer', 'handleMessage', dtoType, message.offset],
-              data,
-            });
+            if (!process.env.DANGEROUS_SKIP_COMMIT_TO_KAFKA || handlerCount) {
```
Comment (Strato) [inline src/consumer.js]: `if (process.env.DANGEROUS_SKIP_COMMIT_TO_KAFKA !== 'true' || handlerCount) {`

Une variable d'environnement, c'est toujours un string.

Comment (FlorianSara) [reply src/consumer.js]: fixé

**Pull request merged**